src/nginx/t/auth_asymmetrickey.t

```diff
@@ -90,7 +103,7 @@ http {
 }
 EOF
 
-my $es256_pubkey = <<'EOF';
+my $pubkeys = <<'EOF';
```

Comment:
add comment to mention two pubkeys are used, their "alg" are different.  ESP will based on "alg" field to choose the public key.

Reply:
Done.